src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -69,10 +69,14 @@ public function validate(mixed $entity, Constraint $constraint)
         }
 
         if ($constraint->em) {
-            $em = $this->registry->getManager($constraint->em);
-
-            if (!$em) {
-                throw new ConstraintDefinitionException(sprintf('Object manager "%s" does not exist.', $constraint->em));
```

Comment:
I removed the handling of `null` here, since it is not really supposed to happen. Let me know if that's wrong.

Reply:
that's totally fine, as it can only happen with a bad mock.